R/connect_inbo_dbase.R

```diff
@@ -58,13 +60,38 @@ connect_inbo_dbase <- function(database_name, autoconvert_utf8 = TRUE) {
     }
 
     # connect to database
-    conn <- dbConnect(odbc(),
-                      driver = sql_driver,
-                      server = server,
-                      port = 1433,
-                      database = database_name,
-                      encoding = encoding,
-                      trusted_connection = "YES")
+    tryCatch(
+        conn <- dbConnect(odbc(),
+                          driver = sql_driver,
+                          server = server,
+                          port = 1433,
+                          database = database_name,
+                          encoding = encoding,
+                          trusted_connection = "YES"),
+        error = function(e) {
+            if (grepl("connection to SQL Server", e)) {
```

Comment:
You could write this as `assert_that(!grepl("connection to SQL Server", e), msg = ...)` Which avoids the if statements.

```
assert_that() # connection
assert_that() # login
stop(e)
```